compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -10295,7 +10295,7 @@ OMR::Z::TreeEvaluator::arraycmpHelper(TR::Node *node,
       {
       // An empirical study has showed that CLC is faster for all array sizes if the number of bytes to copy is known to be constant
       if (!node->getChild(2)->getOpCode().isLoadConst())
-          return TR::TreeEvaluator::arraycmpSIMDHelper(node, cg, compareTarget, ificmpNode, needResultReg, return102);
```

Comment:
Checking out the [arraycmpSIMDHelper](https://github.com/eclipse/omr/blob/104d1f60f5a242078a88364fe5b78f54ef544db2/compiler/z/codegen/OMRTreeEvaluator.hpp#L1018-L1024), default last parameter is set to `false`, We would not need to set last parameter. 

Is there specific reason why you are explicitly setting it to `false` ? Or maybe I am missing something as I do see you added that in other places as well.